frontend/src/component/common/DialogFormTemplate/ConfigButtons/shared.styles.tsx

```diff
@@ -12,11 +12,27 @@ const visuallyHiddenStyles = {
     whiteSpace: 'nowrap',
 };
 
+const padding = (theme: Theme) => theme.spacing(1.5);
+
+export const StyledPopover = styled(Popover)(({ theme }) => ({
+    '& .MuiPaper-root': {
+        borderRadius: `${theme.shape.borderRadiusMedium}px`,
+        paddingInline: 0,
+        paddingTop: padding(theme),
+        display: 'flex',
+        flexDirection: 'column',
+
+        gap: theme.spacing(1),
+        maxHeight: '70vh',
+    },
+}));
+
 export const StyledDropdownSearch = styled(TextField, {
     shouldForwardProp: (prop) => prop !== 'hideLabel',
 })<{ hideLabel?: boolean }>(({ theme, hideLabel }) => ({
+    paddingInline: padding(theme),
```

Comment:
```suggestion
    paddingInline: theme.spacing(dropdownPadding),
```
Wrapping it in a function is an unnecessary misdirection. Every closure is another thing to keep on the "stack" in your head when reading this. Does this need to always match the '& .MuiInputBase-root'? Then maybe setting `const dropdownPadding = 1.5` is a solution? It's an alternative to a CSS variable.

Reply:
good call! Addressed in d8e97fa08aed7280ee0bc450630d110c8c4b9d31. Of course, me being me, I still like CSS variables, but hey .. I can do that in my own time 😄